components/compositing/windowing.rs

```diff
@@ -192,4 +192,7 @@ pub trait WindowMethods {
     /// will want to avoid blocking on UI events, and just
     /// run the event loop at the vsync interval.
     fn set_animation_state(&self, _state: AnimationState) {}
+
+    /// Print Microdata on the Console or write to file
+    fn print_microdata(&self, _data: String, _datatype: String) {}
```

Comment:
This doesn't actually have to print anything when implemented; we should call it `microdata_available` instead, and document what the arguments present.

Reply:
Actually, the name of the function is misleading we are actually writing the output to a file ( .vcf). We'll change the comment.